### PR TITLE
DOI metadata subject area classification enrichment for Data analysis Spec

### LINKED ISSUE #37: DOI metadata enrichment  (subject area classification) for Data analysis (JR)
[<u>DOI metadata enrichment</u>](https://docs.google.com/document/d/1QBFoJlVo86XDL4u78X-ckerOw0CPqQi6G4v3XQwNwLQ/edit#)

- Requirement for MDC

- Product spec is done and ready for the tech spec hand over

- Hand over date: 22\/01\/2021

- Development lead: \[TBC\]

<p></p>

<p></p>

**SP5: Adapting our processes and workflows to meet our members needs**

---

### PR DESCRIPTION
**Pull request by kjgarza**
## Purpose

This is a new product specification. 
To review go to **Files changed** and click **Display the rich Diff**

addresses: https://github.com/datacite/datacite/issues/1154

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Reviewer, please remember to:

- Be humble in the language and feedback you give, ask don't tell.
- Consider using positive language as opposed to neutral when offering feedback. This is to avoid the negative bias that can occur with neutral language appearing negative.
- Ensure you give reasons for the changes you are proposing.

### CODE REVIEW

**Review by mfenner — COMMENTED** [review]
This will be discussed in a team call this week instead.

File: product_specifications/ELmjUzA-7/ELmjUzA-7.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- [ ] As a repository Manager from a single discipline repository, I would like to get suggestions about my repository discipline based on my re3DataId. 
+- [ ] As Staff, I would like that all discipline DOI metadata enrichment is performed via the  OECD subject classification so we can analyze DOI metadata using this controlled vocabulary.
+- [ ] As Staff, I would like to [restrict the access](https://github.com/datacite/lupo/blob/5c9de77d427f7accf29057538dd52b89fb82c82a/app/serializers/datacite_doi_serializer.rb#L247) to discipline enriched DOI metadata fields for non-authenticated users, so that do not confuse our repositories. 
+- [ ] As Staff, I would like to restrict the access to discipline enriched DOI metadata fields to be displayed only via the GraphQL API, so that do not confuse our repositories.
+
+## Non Functional Requirements
+
+- [ ] As a user, I expect the UI to add/remove/update discipline metadata to the repository to follow the default Fabrica style and usability.
+- [ ] As a user, I expect only the client admin to add/removing/updating discipline metadata to the repository.
```
Comment (kjgarza) [inline product_specifications/ELmjUzA-7/ELmjUzA-7.md]: add staff admin

File: product_specifications/ELmjUzA-7/ELmjUzA-7.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+further details: https://docs.google.com/document/d/1sJUQ_IemyPp0CktKXv_tfa7PhHcZ1NCBNUZwAv0iwCc/edit#
+
+## User Stories
+
+### CRITICAL
+- [ ] As an MDC partner, I would like to get the DOIs metadata from single discipline repositories enriched with disciplinary metadata from the repository so that I can carry disciplinary analysis.
+- [ ] As a repository Manager from a single discipline repository, I would like to get suggestions about my repository discipline based on my re3DataId. 
+- [ ] As Staff, I would like that all discipline DOI metadata enrichment is performed via the  OECD subject classification so we can analyze DOI metadata using this controlled vocabulary.
+- [ ] As Staff, I would like to [restrict the access](https://github.com/datacite/lupo/blob/5c9de77d427f7accf29057538dd52b89fb82c82a/app/serializers/datacite_doi_serializer.rb#L247) to discipline enriched DOI metadata fields for non-authenticated users, so that do not confuse our repositories.
```
Comment (kjgarza) [inline product_specifications/ELmjUzA-7/ELmjUzA-7.md]: restrict other APIs, it should work only in graphQL

**Review by kjgarza — COMMENTED** [review]
comments from meeting https://docs.google.com/document/d/1xYdph4Yg61ZzuFcsUuoxtTPOmYuLC-APLQ3RP16byy0/edit

**Pull request merged**